docs/howto/howto_macros.md

```diff
@@ -72,20 +72,28 @@ The `@mockable[A]` generates _capability tags_ and _mock layer_ into annotated o
 import zio.test.mock.mockable
 
 @mockable[AccountObserver.Service]
-object AccountObserverMock {
+object AccountObserverMock
+```
 
-  // below will be autogenerated
-  sealed trait Tag[I, A] extends Method[Has[AccountObserver.Service], I, A] {
-    def envBuilder: URLayer[Has[Proxy], Has[AccountObserver.Service]] = ???
-  }
+Will result in:
+
+```scala
+import zio.{ Has, UIO, URLayer, ZLayer }
+import zio.test.mock.{ Mock, Proxy }
+
+object AccountObserverMock extends Mock[Has[AccountObserver.Service]] {
 
-  object ProcessEvent extends Tag[Unit, Unit]
+  object ProcessEvent extends Effect[AccountEvent, Nothing, Unit]
+  object RunCommand   extends Effect[Unit, Nothing, Unit]
 
-  private val envBuilder: URLayer[Has[Proxy], AccountObserver] =
-    ZLayer.fromService(invoke =>
-      new AccountObserver.Service {
-        def processEvent(event: AccountEvent) = invoke(ProcessEvent, event)
+  val compose: URLayer[Has[Proxy], AccountObserver] =
+    ZLayer.fromServiceM { proxy =>
+      withRuntime.map { rts =>
+        new AccountObserver.Service {
+          def processEvent(event: AccountEvent) = proxy(ProcessEvent, event)
+          def runCommand(): UIO[Unit]           = proxy(RunCommand)
```

Comment:
Usually, empty brackets are added to methods without parameters that are doing side-effects. If we have `UIO` here there are no side-effects. At least there shouldn't be. However, I'm not sure what's the convention for effectful methods